### ISSUE TITLE
Site 18-Waldoboro - Late Fee Setting Access

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#1948]
In GitLab by @kdjstudios on Nov 10, 2020, 08:45

**Submitted by:** <becki.dibert@answernet.com>

**Helpdesk:** https://www-servicedesk.answernet.com/profiles/ticket/2020-11-03-48917

**Server:** Internal

**Client/Site:** Waldaboro

**Account:** NA

**Issue:** 

I do not have this option, can you assist me with switching this as the late fee being added when it should not be is causing a lot of confusion for my clients.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 10, 2020, 08:46

This is in regards to the late fee calculation setting on the site edit page. Which TO my thought was available to anyone who could view the edit site page. We will want to review this further and determine who needs access to this feature/option and possible update SAB.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 10, 2020, 10:27

assigned to @kdjstudios

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 11, 2020, 13:46

I found where in the code this was only being displayed for Admins, while this should be displayed for all users who can access the site edit page. I updated this and also found while in the progress where we are missing the o7 access too. I can not find that ticket so bundling the update here. NOTE: Once I find it I will link to this ticket here also. EDIT: #1918
